### DISCUSSION TITLE
Lc0 and fine tuning

### DISCUSSION THREAD

**Discussion by StepHaze** [LeelaChessZero/lc0#1964] [Q&A]
Hello,
is it possible what I would like to do?
I'd like to take a Lc0's pre-trained model and train it on new data (1000 chess games of mine), so it will play in my style and my strength?

Thanks!

**Comment by brianprichardson** [comment]
It is not possible as a practical matter.
Several hundred thousand games would be required for a net to begin to learn a particular style.

**Comment by loneicewolf** [comment]
it was a interesting question though, @StepHaze

**Comment by StepHaze** [comment]
> It is not possible as a practical matter. Several hundred thousand games would be required for a net to begin to learn a particular style.

Why does it require so many games?

**Comment by CallOn84** [comment]
> > It is not possible as a practical matter. Several hundred thousand games would be required for a net to begin to learn a particular style.
> 
> Why does it require so many games?

He might be referring to distil training, but I'm unsure. It would require a lot of games because it needs a lot of positions to learn and understand how to handle certain positions, and 1k games probably won't provide many positions for Leela to learn from.
